agent/service.go

```
@@ -117,11 +125,13 @@ func (ts *TaskService) Create(requestCtx context.Context, req *taskAPI.CreateTas
 		return nil, err
 	}
 
-	// TODO replace with proper drive mounting once that PR is merged. Right now, all containers in
-	// this VM start up with the same rootfs image no matter their configuration
-	err = bundleDir.MountRootfs("/dev/vdb", "ext4", nil)
-	if err != nil {
-		return nil, err
+	// TODO: Pass new mount path here
+	drive, ok := ts.driveHandler.GetDrive(strings.TrimSpace(extraData.DriveID))
+	if ok {
```

Shouldn't we return an error if `!ok`?